src/release-pr.ts

```diff
@@ -198,7 +228,7 @@ export class ReleasePR {
     return Promise.resolve(undefined);
   }
 
-  static tagSeparator(): string {
+  tagSeparator(): string {
```

Comment:
This is technically breaking

Reply:
I think it's okay given the dependency counts on npm to safely move this.